lib/os_places_api/client.rb

```diff
@@ -6,13 +6,7 @@ def initialize(token_manager)
       @token_manager = token_manager
     end
 
-    def locations_for_postcode(postcode, update: false)
-      postcode = PostcodeHelper.normalise(postcode)
-
-      if (record = Postcode.find_by(postcode: postcode)) && !update
-        return build_locations(record["results"])
-      end
-
+    def get_api_response(postcode)
```

Comment:
Can we move this below the `private` line, to make it private? We only really want to expose `update_postcode` and `locations_for_postcode`.